dialects.md

```diff
@@ -385,24 +384,108 @@ validations:
 In the SHACL mapping, the generation of the `sh:maxCount 1` assertion would be omitted.
 By default the nested nodes will be stored in the graph without any particular order. The `sorted` boolean property can be used to enforce ordering in the nested nodes. In this case the generated graph will keep the nodes in an ordered RDF collection.
 
-
 ## Nesting
+In AML it is possible to add an extra layer of nesting between nodes in dialect instances and map the resulting key-value pairs 
```

Comment:
No se si me queda claro. Por ahi podria algo mas como:
In AML is possible to use a particular mapping value as key of the map for objects collections, instead of use a sequence structure. This will mean that the mapping used as key will be implicit, deduced syntactically from the key.
Even, if you want a list of key:value values for object with two properties (simply model the map-entry) you can configure implicit properties as key and value as well.

Reply:
Es difícil de explicar abstractamente, me dio el mismo feedback Agus B. A mí tampoco me termina de quedar clara la alternativa que propones. Charlamos la explicación de alto nivel y lo validamos con alguien que no tenga contexto del feature pero sí de AML?

Reply:
Si me parece bien, bajemos una defincion y validarla. No se si alguien que no tenga nada de contexto (necesitas explicarle otras cosas) tal vez alguien que recien arranca con el tema (Imaginaba en un chico de Gus que se esta metiendo de apoco con AML).